go.mod

```diff
@@ -3,15 +3,18 @@ module github.com/peggyjv/gravity-bridge
 go 1.15
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.3
+	github.com/cosmos/cosmos-sdk v0.46.0
```

Comment:
0.46.3?

Reply:
I have matched the dependency in the parent folder to be the same as the module dependency for consistency.

We could upgrade them in a separate PR